tests/integration/targets/ec2_key/aliases

```diff
@@ -1 +1,6 @@
+# reason: missing-dependency
+# We need either the openssl binary, pycrpto, or a compiler on the Py36 and Py38
+# Zuul nodes
```

Comment:
You can install this in CI by creating a top-level bindep.txt file with the following:

  openssl [test platform:rpm]
  
for example. https://docs.opendev.org/opendev/bindep/latest/ gives a few more examples.

Reply:
I've tried adding that file and it still failes:
https://dashboard.zuul.ansible.com/t/ansible/stream/08ad48a55d024770bc7e4766fb3caa82?logfile=console.log